common.tf

```diff
@@ -48,3 +48,24 @@ data "aws_iam_policy_document" "waggle_dance_glue_policy" {
     ]
   }
 }
+
+data "aws_secretsmanager_secret" "datadog_key" {
+  name  = var.datadog_key_secret_name
+}
+
+data "aws_secretsmanager_secret_version" "datadog_key" {
+  secret_id = data.aws_secretsmanager_secret.datadog_key.id
```

Comment:
`datadog_key_secret_name` is set as "" by default. what is the behaviour of this code when a valid secret is not created in the secrets manager ? 

Does it fail the deployment ? I think we want to make this an optional feature.

Reply:
made it optional.